test/src/main/java/org/apache/accumulo/test/functional/BackupManagerIT.java

```diff
@@ -52,13 +52,13 @@ public void test() throws Exception {
       // wait for 2 lock entries
       do {
         UtilWaitThread.sleep(100);
-        children = writer.getChildren(root + Constants.ZMANAGER_LOCK);
+        String path = root + Constants.ZMANAGER_LOCK;
+        children = ZooLock.validateAndSortChildrenByLockPrefix(path, writer.getChildren(path));
       } while (children.size() != 2);
-      Collections.sort(children);
       // wait for the backup manager to learn to be the backup
       UtilWaitThread.sleep(1000);
       // generate a false zookeeper event
-      String lockPath = root + Constants.ZMANAGER_LOCK + children.get(0);
+      String lockPath = root + Constants.ZMANAGER_LOCK + "/" + children.get(0);
```

Comment:
Thanks for catching this. I had run this test and fixed the same bug, which I introduced when I replaced the text with the constant, but in managing multiple pull requests I must have lost the change.